core/src/main/java/org/jdbi/v3/core/Jdbi.java

```diff
@@ -366,11 +373,18 @@ public Jdbi registerExtension(ExtensionFactory<?> extensionFactory)
         return this;
     }
 
-    public <C extends ExtensionConfig<C>> Jdbi configureExtension(Class<C> configClass, Consumer<C> consumer) {
+    public <C extends ExtensionConfig<C>> Jdbi configureExtension(Class<C> configClass, Consumer<C> consumer)
+    {
         config.extensionRegistry.configure(configClass, consumer);
         return this;
     }
 
+    public <C extends ExtensionConfig<C>> Jdbi configure(Class<C> configClass, Consumer<C> consumer)
+    {
+        registerExtension(new ConfigOnlyExtension<C>(configClass));
```

Comment:
Repeated calls for the same config class will register duplicate (and useless) extensions.